documentation/modules/exploit/multi/http/atlassian_crowd_pdkinstall_plugin_upload_rce.md

```diff
@@ -0,0 +1,91 @@
+## Vulnerable Application
+
+Atlassian Crowd 2.1.x prior to 3.0.5, 3.1.x prior to 3.1.6, 3.2.x prior to 3.2.8, 3.3.x prior to 3.3.5, and 3.4.x prior to 3.4.4
+had an incorrect installation of the pdkinstall development plugin, which allows unauthenticated remote attackers to upload and install
+arbitrary plugins on vulnerable Atlassian Crowd and Atlassian Crowd Data Center installations via a POST request to the
+`/<crowd install base>/admin/uploadplugin.action` page.
+
+Successful exploitation results in remote code execution as the user running the Atlassian Crowd server.
+
+To set up a vulnerable installation:
+1. Grab a copy of Windows, any version will do but I tested on Windows 10.
+1. Grab a copy of the 3.0.3 version of Atlassian Crowd from https://product-downloads.atlassian.com/software/crowd/downloads/atlassian-crowd-3.0.3.zip
+1. Grab a copy of the Java 8 JDK from https://download.java.net/openjdk/jdk8u41/ri/openjdk-8u41-b04-windows-i586-14_jan_2020.zip
+1. Unzip the JDK to a target directory.
+1. Edit `JAVA_HOME` environment variable and set it to the location where you extracted the Java 8 JDK.
+1. Update the `PATH` environent variable to include a path to the same location as `JAVA_HOME`, but with `\bin` at the end of it.
```

Comment:
```suggestion
1. Update the `PATH` environment variable to include a path to the same location as `JAVA_HOME`, but with `\bin` at the end of it.
```

Reply:
Nice spotting, will fix this up 👍

Reply:
Fixed with https://github.com/rapid7/metasploit-framework/pull/15501/commits/0d7d5ab93f5b9bd8afcefdd1863054164922084b